BendingSpoons/fastlane
# [spaceship] Support manual app release

### Checklist

### Motivation and Context


This PR adds support for `appStoreVersionReleaseRequests`, enabling fastlane users to release an app update that has been approved by Apple and for which the release type is set to manual release.

### Description


The PR adds a new `AppStoreVersionReleaseRequest` model to the App Store Connect API interface and a dedicated `create_app_store_version_release_request` method in the `AppStoreVersion` model.

### Testing Steps


I've written two unit tests to cover the new logic of the Connect API Tunes client and the expected API response. They can be run individually as follows:
```
bundle exec rspec ./spaceship/spec/connect_api/models/app_store_version_release_request_spec.rb
bundle exec rspec ./spaceship/spec/connect_api/tunes/tunes_client_spec.rb
```
